packages/cmf/__tests__/selectors/toJS.test.js

```diff
@@ -0,0 +1,42 @@
+import Immutable from 'immutable';
+import toJS from '../../src/selectors/toJS';
+
+describe('toJS', () => {
```

Comment:
miss a test where the state get mutated between two calls, and the memoized function should invalidate its inner cache, to return the new proper result.